tracking_manager/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+- In setting > models: select a model
+- Check "Apply custom tracking on fields", all the non readonly, related, computed fields will be tracked.
+- Button or smart button "Tracked Fields" allow to activate / deactivate custom_tracking on each field
+- By default, all sub fields of a one2many field are tracked if they changed. To manage the sub fields, you need to activate "Apply custom tracking on fields" on the one2many model. Button "One2many related models" is a shortcut for these models.
```

Comment:
This content looks obsolete after the recent changes, perhaps.

It's probably more appropriate to have this content in CONFIGURE.rst.

Reply:
@Kev-Roche can you help me to update the doc and add some screenshot ?

Reply:
I did it